There is almost always a signal 6 or 11 on exit; not sure why, but let's
ignore them for now.

--- origsrc/kdelibs-4.1.4/kdeui/util/kcrash.cpp	2009-01-06 17:37:07.000000000 -0600
+++ src/kdelibs-4.1.4/kdeui/util/kcrash.cpp	2009-01-25 16:12:01.328625000 -0600
@@ -210,7 +210,7 @@ closeAllFDs()
 void
 KCrash::defaultCrashHandler (int sig)
 {
-#ifdef Q_OS_UNIX
+#if defined(Q_OS_UNIX) && !defined(Q_OS_CYGWIN)
   // WABA: Do NOT use kDebug() in this function because it is much too risky!
   // Handle possible recursions
   static int crashRecursionCounter = 0;
